2-mandatory/4-stocks.js

```diff
@@ -49,6 +54,9 @@ function getAveragePrices(closingPricesForAllStocks) {
 */
 function getPriceChanges(closingPricesForAllStocks) {
     // TODO
+    return closingPricesForAllStocks.map((priceArr) => {
+      return Number((priceArr[priceArr.length - 1] - priceArr[0]).toFixed(2));
+    });
 }
```

Comment:
Great job on these two functions.
There is one more left `highestPriceDescriptions`, please give it another try.